app/src/components/widgets/charts/cumulativeTrendChart/chartjsConfig.js

```diff
@@ -146,6 +146,9 @@ const getChartOptions = (widget, options) => {
           const value = dataset.data[tooltipItem.index];
           const totalValue = totalDataset.data[tooltipItem.index];
           const percentageValue = -(-value / totalValue * 100).toFixed(2);
+          if (!value) {
```

Comment:
Please do this check immediately after [this](https://github.com/reportportal/service-ui/pull/1880/files#diff-17259b00e31d9263f7d5cae73a8bf19fL146) line to prevent unnecessary calculations.